client/src/components/History/CurrentHistory/HistoryOperations/Operations.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+    <section>
+        <b-dropdown
+            no-caret
+            size="sm"
+            variant="link"
+            class="rounded-0"
+            toggle-class="text-decoration-none rounded-0"
+            data-description="history action menu"
+            :disabled="!hasMatches">
```

Comment:
I'm not 100% sure about disabling this when there are no current matches. It may happen that you hide all items in history and then you want them back. The only way to bring them back is by explicitly filtering for hidden items otherwise you cannot access the `Unhide all hidden content`.
This probably won't be an issue when we display the summary of active/deleted/hidden items somewhere like we do in the previous history panel and use this info to toggle the actions of this 'with entire history' cog menu.

Reply:
Good catch, will remove it.